src/tests/architect_tests/utils.py

```diff
@@ -95,50 +87,47 @@ def create_entity_date_df(
     state_two,
     label_name,
     label_type,
-    label_timespan
+    label_timespan,
```

Comment:
The trailing comma here at the last argument of the function is kinda confusing me.

Reply:
Adding/removing/changing around arguments only requires moving around lines when it's done this way. I'm used to seeing it in lists/dictionaries but not arguments as much, though I don't think I mind.

Reply:
I was wrong! The trailing comma makes a lot of sense when we look at the git diff.